python_modules/dagster/dagster/_legacy/__init__.py

```diff
@@ -10,8 +10,6 @@
     PartitionSetDefinition as PartitionSetDefinition,
     PipelineDefinition as PipelineDefinition,
     PresetDefinition as PresetDefinition,
-    ScheduleExecutionContext as ScheduleExecutionContext,
```

Comment:
What's our current policy with eliminating symbols from legacy? Can we just delete them at will?

Reply:
Yeah stuff in `dagster._legacy` is only used internally, so it can be deleted once all the references are gone.